app/src/main/kotlin/com/metrolist/music/ui/component/SettingsSleepTimerDialog.kt

```
@@ -0,0 +1,659 @@
+@Composable
+fun SleepTimerTimePickerDialog(
+    title: String,
+    onDismiss: () -> Unit,
+    onConfirm: (String) -> Unit,
+    initialTime: String = DEFAULT_START,
+) {
+    val timeFormatter = DateTimeFormatter.ofPattern("HH:mm")
+    val initialLocalTime =
+        try {
+            LocalTime.parse(initialTime, timeFormatter)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)